faunadb-common/src/main/java/com/faunadb/common/Connection.java

```diff
@@ -320,7 +320,7 @@ private String mkUrl(String path) throws MalformedURLException {
     return new URL(faunaRoot, path).toString();
   }
 
-  private void logSuccess(Request request, Response response) {
+  private void logSuccess(Request request, Response response) throws IOException {
```

Comment:
We should not throw an exception when logging this. Specially because it's a debug message.